experimenter/experimenter/segments/__init__.py

```diff
@@ -31,10 +49,13 @@ def _load_segments(
 
         config_collection = None
         if segment_data is None:
-            config_collection = ConfigCollection.from_github_repos(
-                [
-                    "https://github.com/mozilla/metric-hub",
-                ]
+            repo_git_dir = Path("/experimenter/experimenter/segments/metric-hub-main")
```

Comment:
@jaredlockhart This path should be valid in all environments, right?

Reply:
We probably want this to be a setting in settings.py so that it can be configurable.

Reply:
Oh yea good point. It is in settings already (`METRIC_HUB_SEGMENTS_PATH_DEFAULT` though value is slightly different now), but I think this got a bit jumbled when we moved things around in the last PR. Pretty sure we can just modify that setting and use it here.